packages/legacy-workbench/src/js/angular/guides/steps/core/plugin.js

```diff
@@ -12,6 +12,7 @@ const BASIC_STEP = {
     onNextValidate: () => Promise.resolve(true),
     onPreviousClick: undefined,
     skipPoint: false,
+    skipLabel: 'skip.btn',
```

Comment:
Maybe we could rename skipLabel to skipButtonLabel so it’s more obvious what it refers to.